boot/freeldr/freeldr/ntldr/wlregistry.c

```diff
@@ -536,10 +537,17 @@ WinLdrScanRegistry(IN OUT PLIST_ENTRY BootDriverListHead,
     /* Get the Name Group */
     BufferSize = 4096;
     GroupNameBuffer = FrLdrHeapAlloc(BufferSize, TAG_WLDR_NAME);
+    if (!GroupNameBuffer)
+    {
+        return;
```

Comment:
This is most likely a fatal failure. Should probably just call FrLdrBugCheck or something, but at least add an ERR() here.

Reply:
Adding `ERR()`s was previously refused (twice)! Simply returning matches the other failure cases.
If you want more, in this PR or a follow-up, please get explicit approval from @HBelusca and @learn-more.